modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSMessageStoreImpl.java

```diff
@@ -372,16 +373,22 @@ public void storeMessages(List<AndesMessage> messageList) throws AndesException
         Connection connection = null;
         PreparedStatement storeMetadataPS = null;
         PreparedStatement storeContentPS = null;
+        PreparedStatement storeExpiryMetadataPS = null;
 
         try {
 
             connection = getConnection();
             storeMetadataPS = connection.prepareStatement(PS_INSERT_METADATA);
             storeContentPS = connection.prepareStatement(PS_INSERT_MESSAGE_PART);
+            storeExpiryMetadataPS = connection.prepareStatement(PS_INSERT_EXPIRY_DATA);
```

Comment:
it seems we are always creating this prepared statement which is redundant don't u think?